code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -34,11 +35,12 @@
 	use_cyborg_cell = 1
 
 /obj/item/weapon/gun/energy/disabler
-	name = "disabler"
+	name = "pulse disabler"

```

Comment:
Do you mind renaming it to something that isn't associated with deathsquad related weaponry?